app/models/concerns/indexable.rb

```diff
@@ -33,13 +33,15 @@ def set_published_at
       unless published_at
         if defined?(metadata) && metadata && metadata['date']
           begin
-            update_column(:published_at, metadata['date'])
-          rescue ActiveRecord::StatementInvalid
-            update_column(:published_at, created_at)
+            self.published_at = Date.parse(metadata['date'])
+          rescue StandardError
```

Comment:
I changed it to 'StandardError' in order to rescue from both the `ActiveRecord::StatementInvalid` and the `Argument Error: Invalid Date` but then I extracted the `save!` out of the rescue block (the `ActiveRecord::StatementInvalid` shouldn't happen anymore anyway). I forgot to switch back and I'm going to do that right now ;)